### ISSUE TITLE
Does `amg::rebuild` work?

### ISSUE THREAD

**Issue by alsam** [ddemidov/amgcl#208]
Hello Denis,
I've stumbled upon the issue [Rebuild the preconditioner](https://github.com/ddemidov/amgcl/issues/101) but as it was closed in 2019 I wonder if it works now.

**Comment by ddemidov** [comment]
Yes, see https://amgcl.readthedocs.io/en/latest/components/preconditioners.html#_CPPv4N5amgcl3amg6params13allow_rebuildE
